ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```
@@ -169,6 +177,10 @@ images:
   dockerfile_path: Dockerfile
   from: cuda-c9s-python-3.9
   to: cuda-rstudio-c9s-python-3.9
+- context_dir: amd/c9s-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-c9s-python-3.9
+  to: amd-c9s-python-3.9
```

```suggestion
  to: amd-c9s-python-3.9
- context_dir: jupyter/minimal/ubi9-python-3.9
  dockerfile_path: Dockerfile
  from: amd-c9s-python-3.9
  to: amd-jupyter-minimal-c9s-python-3.9
```
based on the suggestion, we could actually us the minimal for testing